### DISCUSSION TITLE
Question about Environment module

### DISCUSSION THREAD

**Discussion by kupe-dono** [semaphoreui/semaphore#842] [Q&A] [answered]
Hello!

I just have few questions about the environment module in ansible-semaphore. I don't understand the purpose of this module and how to use it and I found very little information about it on the web or in your wiki documentation.

My questions are as follows: 
- Is the module optional? (can I just put the brackets {} and let them empty?)
- Is it possible to provide few use cases with this module in your documentation?

Thank you so much for your great work! 

Jim

**Comment by fiftin** [comment, marked as answer]
Hi @kupe-dono,

1) Yes, it can be `{}`
2) You are right, I will add. Environment is passed to `ansible-playbook` via `--extra-vars` argument.

**Comment by kupe-dono** [comment]
Hi @fiftin,
Thank you for your help! Have a nice day :)
